src/foam/lib/json/StringArrayParser.java

```diff
@@ -12,22 +12,28 @@
   extends ProxyParser
 {
   public StringArrayParser() {
-    super(new Seq1(3,
-      new Whitespace(),
-      new Literal("["),
-      new Whitespace(),
-      new Repeat(
-        new StringParser(),
-        new Seq0(new Whitespace(), new Literal(","), new Whitespace())),
-      new Whitespace(),
-      new Literal("]")));
+    super(
```

Comment:
Why is this needed?

Reply:
if the StringArrayParser can not parse null, the change will not be merged to memory, when system reboot